src/traces/box/index.js

```diff
@@ -10,10 +10,18 @@
 'use strict';
 
 var Plotly = require('../../plotly');
-var d3 = require('d3');
-var isNumeric = require('fast-isnumeric');
 
-var boxes = module.exports = {};
+var boxes = {
```

Comment:
I used to `exports` object directly instead in the other trace modules (e.g. https://github.com/plotly/plotly.js/blob/master/src/traces/histogram/index.js#L39).

I'd vote for keeping the same pattern, if you don't mind.